src/components/LoadingWithoutModal/index.js

```diff
@@ -28,7 +28,7 @@ const styles = StyleSheet.create({
     borderRadius: 10
   }),
   text: {
-    color: COLORS.white,
+    color: '#fff',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been hardcoded to `'#fff'`, which might make it difficult to maintain consistency across the application. It would be better to use a color constant from the `colors` module.
```diff
- color: '#fff',
+ color: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->